docs/about/history/index.html

```diff
@@ -0,0 +1,85 @@
+	next steps for the Web of Things</a> were discussed and an Interest Group was formed with the aim to identify use cases 
+	and requirements for web standards to enable open markets of services that build on top of the Internet of Things, 
+	in combination with the Web of Data.
+	</p>
+	<p>
+	<a target="_blank" href="https://www.w3.org/WoT/IG/wiki/F2F_meeting:_20-22_April_2015_in_Munich">The first Face to Face 
+	meeting of the Interest Group</a> happened in April 2015 in Munich, Germany.
+	The group has identified that different task forces such as Thing Description, Architecture and Scripting API were
+	needed in order to produce corresponding deliverables.
+	The Interest Group had <a target="_blank" href="https://www.w3.org/WoT/IG/wiki/F2F_meeting_29-30_October_2015,_Sapporo,_Japan">
+	the first ever WoT PlugFest in Sapporo</a>. 
+	Companies such as Siemens, INSTITUT TELECOM (France), FOKUS, KDDI, ETRI, ACCESS, and Fujitsu verified Thing Description-based
+	 basic interoperability between participating implementations. 
+	Given the success in Sapporo, PlugFest was institutionalized in WoT IG. 
+	In virtually every Face-to-Face meeting thereafter starts with one or two days PlugFest before technical discussion days.
+	</p>
+
+	<!-- <figure>
+	<img src="/WoT/images/Wot-ig-sapporo.jpg" width="100%" alt="First WoT Plugfest in Sapporo">
+	<figcaption style="text-align: center;">First Web of Things Plugfest in Sapporo, Japan.</figcaption>
+	</figure> -->
+
+	<br>
+
+	<p>
+	In December 2016, the W3C WoT Working Group has started its work with its first charter with the aim of 
+	producing 4 specifications, namely Thing Description, Architecture as normative specifications and 
+	Scripting API, Binding Templates as an informative ones.
+	In June 2019, the groups have organized <a target="_blank" href="https://www.w3.org/WoT/ws-2019/"> the Second WoT Workshop</a> 
+	in Munich, Germany to celebrate the finalization of the deliverables of the charter.
+	Indeed, the Working Group has published the promised specifications in early 2020, with 
+	<a target="_blank" href="https://www.w3.org/TR/2020/REC-wot-thing-description-20200409/"> WoT Thing Description</a>
+	 and <a target="_blank" href="https://www.w3.org/TR/2020/REC-wot-architecture-20200409/"> WoT Architecture</a> 
+	 being two brand new W3C Recommendations. 
+	</p>
+
+	<!-- <figure>
+		<img src="/WoT/images/ws2019-Group-photo.png" width="100%" alt="Second WoT Workshop">
+		<figcaption style="text-align: center;">Second W3C WoT Workshop in Munich, Germany</figcaption>
+	</figure> -->
+
+	<br>
+	
+	<p>
+	Since then, the Working Group is working on new specifications such as the WoT Profiles and WoT Discovery, while
+	working on the newer versions of Thing Description, Architecture, Scripting API and Binding Templates.
+	If you are interested in learning about WoT, check our <a href="/WoT/documentation"> documentation page</a>.
```

Comment:
I think this `href` should read differently and without "WoT"

`href="{{'documentation/' | relative_url }}"`

Reply:
Could you test it on your local machine? The behavior of links in netlify is different

Reply:
Your code works locally but fails on netlify (I guess it fails also on the website in the end).

Moreover, my proposed code is how we do it elsewhere and works locally.. also.. and I am pretty sure it works also on netlify... 

Another note: Adding "WoT" prefix to each subpage would not be a good idea if this prefix changes once... we deal with it in `_config.yml`.

Hope this helps.

Reply:
Done the first part. Not sure what you meant by:
`Another note: Adding "WoT" prefix to each subpage would not be a good idea if this prefix changes once... we deal with it in _config.yml.` We can handle that as a separate issue